src/main/scala/org/phenoscape/kb/ingest/bgee/BgeeExpressionToOWL.scala

```diff
@@ -0,0 +1,63 @@
+package org.phenoscape.kb.ingest.bgee
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+import java.io._
+import scala.io.Source
+import org.phenoscape.scowl.Functional._
+
+import org.apache.commons.lang3.StringUtils
+import org.phenoscape.kb.ingest.util.Vocab._
+import org.phenoscape.kb.ingest.util.ExpressionUtil
+import org.phenoscape.kb.ingest.util.OBOUtil
+import org.phenoscape.kb.ingest.util.OntUtil
+import org.phenoscape.scowl._
+import org.semanticweb.owlapi.model.OWLAxiom
+import org.semanticweb.owlapi.model.IRI
+
+object BgeeExpressionToOWL {
+
+  def strToSource(str: String): Source = {
+    io.Source.fromFile(str)
+  }
+
+  def convert(expressionData: Source): Set[OWLAxiom] = expressionData.getLines.flatMap(translate).toSet[OWLAxiom]
+
+  def translate(expressionLine: String): Set[OWLAxiom] = {
+    val items = expressionLine.split("\t", -1)
+    if (items(6).startsWith("absent") || items(6).startsWith("Expression")) {
+      Set.empty
+    } else {
+      val axioms = mutable.Set.empty[OWLAxiom]
+      val geneID = StringUtils.stripToNull(items(0))
+      val anatomicalID = StringUtils.stripToNull(items(2))
+
+      val expression = OntUtil.nextIndividual()
+      axioms.add(Declaration(expression))
+      axioms.add(expression Type GeneExpression)
+
+      val structure = OntUtil.nextIndividual()
+      axioms.add(Declaration(structure))
+
+      val term = (Option(StringUtils.stripToNull(items(2))).filter(_ != "\\").get)
```

Comment:
If you wrap a result with `Option()`, and then call `option.get`, you are just undoing the wrapping. `Option` gives you a typesafe way to mark values that may possibly be null and force you to deal with that possibility. The way you're doing it here will result in a `NullPointerException` if the string is empty. You should pretty much never call `get` on an option.

http://danielwestheide.com/blog/2012/12/19/the-neophytes-guide-to-scala-part-5-the-option-type.html